components/List/src/MDCSelfSizingStereoCell.m

```diff
@@ -165,6 +165,8 @@ - (void)prepareForReuse {
 
   [self mdc_setAdjustsFontForContentSizeCategory:NO];
   [self resetMDCSelfSizingStereoCellLabelProperties];
+
+  [self setNeedsLayout];
```

Comment:
@jpoumailloux , would `[self invalidateCachedLayouts];` do the trick here? Do we need to call `setNeedsLayout`?

Reply:
Calling -invalidateCachedLayouts would not be sufficient. This method is actually already called in -prepareForReuse a couple of lines above, and it still caused truncations of the labels. I've updated the change to remove the call to -invalidateCachedLayouts since calling -setNeedsLayout will also call it (the class overrides -setNeedsLayout to do that).

Reply:
Got it, thanks 👍